content/id/docs/concepts/architecture/controller.md

```diff
@@ -60,7 +60,7 @@ Job adalah sumber daya dalam Kubernetes yang menjalankan a
 {{< glossary_tooltip term_id="pod" >}}, atau mungkin beberapa Pod sekaligus, 
 untuk melakukan sebuah pekerjaan dan kemudian berhenti.
 
-(Setelah [dijadwalkan](../../../../en/docs/concepts/scheduling-eviction/), objek Pod 
+(Setelah [dijadwalkan](/en/docs/concepts/scheduling-eviction/), objek Pod 
```

Comment:
The link cleanup in this PR is nice, but it's currently broken with a 404 error _(see [preview page)](https://deploy-preview-46308--kubernetes-io-main-staging.netlify.app/id/docs/concepts/architecture/controller/#kontrol-melalui-server-api)._ To fix, we can remove 'en' from the start of the link or ideally use the language identifier 'id' of this existing page.

```suggestion
(Setelah [dijadwalkan](/id/docs/concepts/scheduling-eviction/), objek Pod 
```

Reply:
Ah, my bad! Thank you for noticing it!